presage-store-sqlite/src/protocol.rs

```diff
@@ -38,7 +55,21 @@ impl SessionStore for SqliteProtocolStore {
         address: &ProtocolAddress,
         record: &SessionRecord,
     ) -> Result<(), ProtocolError> {
-        todo!();
+        let uuid = address.name();
+        let device_id: u32 = address.device_id().into();
+        let record_data = record.serialize()?;
+        query!(
+            "INSERT INTO sessions ( address, device_id, identity, record ) VALUES ( $1, $2, $3, $4 )",
```

Comment:
As far as I can see, sessions should also be replaced when they already exist (looking at WF).
The same probably also for the various types of keys.

Reply:
Can you give it another look when it's not a draft anymore?

I plan on double checking constraints on tables themselves, and most tables already have `PRIMARY KEY(...) ON CONFLICT REPLACE`.